### PR TITLE
Plugins / i18n: add wporg locale code to the plugin details request

### LINKED ISSUE #5003: Plugins: readme translations do not appear in Calypso
#### Steps to reproduce
0. Set your interface language to French.
1. Starting at URL: https://wordpress.com/plugins/jetpack
2. Scroll down to the Description / Installation / FAQ tabs
3. Since [the readme is translated on WordPress.org](https://fr.wordpress.org/plugins/jetpack/), I would expect each one of these tabs to be translated too.

screen shot 2016-04-26 at 12 20 51 pm

### PR DESCRIPTION
**Pull request by yoavf**
Fixes #5003 
(Now possible thanks to #5688)

### CODE REVIEW

**Comment by yoavf** [comment]
Test:
> 
> Set your interface language to French.
> Starting at URL: https://wordpress.com/plugins/jetpack
> Scroll down to the Description / Installation / FAQ tabs
> Since the readme is translated on WordPress.org, the tabs content be translated too.
> 
>

File: client/lib/wporg/index.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ var debug = require( 'debug' )( 'wporg' ),
 /**
  * Internal dependencies
  */
-var jsonp = require( './jsonp' );
+var jsonp = require( './jsonp' ),
+	i18n = require( 'lib/mixins/i18n' ),
```
Comment (Tug) [inline client/lib/wporg/index.js]: Hi there, i18n usage has just changed recently. please use:
```
	i18n = require( 'i18n-calypso' );
```
instead

**Comment by yoavf** [comment]
Oops - merged the wrong PR :( Pretty confident about the change, so not going to revert. But a post-merge review is still welcome.

**Comment by akirk** [comment]
LGTM

**Pull request merged**